io.github.xiayifang.goldendict_ng.json

```diff
@@ -0,0 +1,182 @@
+          "sha256": "c0f892943208266f9b6543b3ae308fab6284c5c90e627931446fb49b4221a072",
+          "x-checker-data": {
+            "type": "anitya",
+            "project-id": 1868,
+            "url-template": "http://www.oberhumer.com/opensource/lzo/download/lzo-$version.tar.gz"
+          }
+        }
+      ]
+    },
+    {
+      "name": "eb",
+      "config-opts": [
+        "--enable-shared",
+        "--disable-static"
+      ],
+      "cleanup": [
+        "/bin",
+        "/share/doc"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://salsa.debian.org/debian/eb.git",
+          "tag": "debian/4.4.3-9",
+          "x-checker-data": {
+            "type": "git",
+            "tag-pattern": "^debian/([\\d.-]+)$",
+            "sort-tags": false
+          },
+          "commit": "1330077ff49b82a409758e5e1fe4ff95f7b6b146"
+        },
+        {
+          "type": "shell",
+          "commands": [
+            "cp -p /usr/share/automake-*/config.{sub,guess} ."
+          ]
+        }
+      ]
+    },
+    {
+      "name": "xapian-core",
+      "config-opts": [
+        "--disable-sse",
+        "--disable-backend-chert",
+        "--disable-backend-remote",
+        "--disable-documentation"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://oligarchy.co.uk/xapian/1.4.23/xapian-core-1.4.23.tar.xz",
+          "sha256": "30d3518172084f310dab86d262b512718a7f9a13635aaa1a188e61dc26b2288c"
+        }
+      ]
+    },
+    {
+      "name": "libzim",
+      "buildsystem": "meson",
+      "builddir": true,
+      "config-opts": [
+        "--wrap-mode=nodownload",
+        "-Dtest_data_dir=none"
+      ],
+      "cleanup": [
+        "/bin",
+        "/include",
+        "/lib/pkgconfig"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/openzim/libzim.git",
+          "tag": "8.2.0",
+          "commit": "38f67fb7f68c0e150f8494895265566aecc60cce",
+          "x-checker-data": {
+            "type": "anitya",
+            "project-id": 20193,
+            "tag-template": "$version"
+          }
+        }
+      ]
+    },
+    {
+      "name": "goldendict-ng",
+      "buildsystem": "cmake-ninja",
+      "build-options": {
+        "env": {
+          "LD_LIBRARY_PATH": "${LD_LIBRARY_PATH}:/app/lib/:/usr/lib"
+        }
+      },
+      "config-opts": [
+        "-DWITH_XAPIAN=ON",
+        "-DWITH_ZIM=ON",
+        "-DWITH_FFMPEG_PLAYER=ON",
+        "-DWITH_EPWING_SUPPORT=ON"
+      ],
+      "build-commands": [
+        "install -Dm644 redist/icons/goldendict.png ${FLATPAK_DEST}/share/icons/hicolor/256x256/apps/${FLATPAK_ID}.png",
+        "desktop-file-edit --set-key=\"Icon\" --set-value=\"${FLATPAK_ID}\" redist/${FLATPAK_ID}.desktop"
+      ],
+      "cleanup": [
+        "/share/pixmaps"
+      ],
+      "cleanup-commands": [
+        "mkdir -p /app/lib/ffmpeg"
```

Comment:
this should be in the `build-commands`.

Reply:
Movded to `build-commands`

I copied this from others :sweat_smile:

https://github.com/search?q=org%3Aflathub+++++++cleanup-commands+mkdir+-p+%2Fapp%2Flib%2Fffmpeg&type=code